project-base/composer.json

```diff
@@ -49,7 +49,6 @@
         "fzaninotto/faker": "^1.7.1",
         "helios-ag/fm-elfinder-bundle": "^10.0.4",
         "heureka/overeno-zakazniky": "^2.0.6",
-        "incenteev/composer-parameter-handler": "^2.1.3",
```

Comment:
This dependency should be removed even from symfony.lock file (both monorepo/project-base)

Reply:
fixed